pkgs/by-name/tt/tt/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "tt";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "lemnos";
+    repo = "tt";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-oe0gegMINVmbw1UPvSy9G67D4amQTumlrgvb8H+Y4IM=";
+  };
+
+  vendorHash = "sha256-iny0OKInHqoXzYcEtd4f0T/yty5/z3k3yFbDo5yazes=";
+
+  postInstall = ''
+    mv $out/bin/src $out/bin/tt
+  '';
```

Comment:
How about installing a man page with `installManPage`?
https://nixos.org/manual/nixpkgs/unstable/#installshellfiles

Reply:
can I have some guidance about how to do this ? I never did that and can't find how to do it properly for this package :sweat_smile:

Reply:
The source includes man (tt.1.gz), so you just install that.
Since there are many examples in nixpkgs, you can grep to see some of them.